### PR TITLE
Typo Fix in guide/english/computer-hardware/gpu/index.md

### PR DESCRIPTION
**Pull request by iamrajiv**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Comment by iamrajiv** [comment]
@thecodingaviator please review this

**Comment by RandellDawson** [comment]
@iamrajiv Please understand we are still reviewing many other earlier PRs.  It can take a while to get to these.

Thank you.

File: guide/english/computer-hardware/gpu/index.md [hunk 1]
```diff
@@ -1,84 +1,80 @@
...
 
-The GPU was originally used mainly for 3D game rendering to improve your resolution and framerate. Now these capabilities are being harnessed more broadly to improve computational workloads in many areas; for example financial modeling, cutting-edge scientific research and oil and gas exploration. GPU's are also used as a resource for bitcoin mining, since they are able to run repetitive tasks easily without straining the resources of the CPU, which allows you run an Operating System on the computer with a low end CPU while still being able to bitcoin mine using the GPU
+The GPU was originally used mainly for 3D game rendering to improve your resolution and framerate. Now, these capabilities are being harnessed more broadly to improve computational workloads in many areas; for example financial modeling, cutting-edge scientific research and oil and gas exploration. GPU's are also used as a resource for bitcoin mining since they are able to run repetitive tasks easily without straining the resources of the CPU, which allows you run an Operating System on the computer with a low-end CPU while still being able to bitcoin mine using the GPU
```
Comment (RandellDawson) [inline guide/english/computer-hardware/gpu/index.md]: Needs a period at the end of the last sentence after "GPU".

**Review by RandellDawson — CHANGES_REQUESTED** [review]
See my period suggestion.  Also, please do not remove the table of contents?  If anything, it would be better to update it to include other sections.

File: guide/english/computer-hardware/gpu/index.md (hunk 1 above)
Comment (iamrajiv) [reply guide/english/computer-hardware/gpu/index.md]: @RandellDawson thanks for pointing out

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**